src/pkg/terminator/eventkind.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package terminator
 
-type EventKind string
+type EventKind = string
```

Comment:
Can you describe the semantic difference between these two ways of describing the type? I'm not very familiar with it.

Reply:
```go
type EventKind string // Defines a new type whose content is a `string`.
type EventKind = string // Defines an alias for the `string` type, i.e. `EventKind` and `string` values can be used interchangeably.
```

The switch made things a easier when including the event kind description in the webhook notification.